packages/channels-app/src/screens/steps/routes/ChannelRoutes.tsx

```diff
@@ -0,0 +1,243 @@
+import {
+  Box,
+  Button,
+  Chip,
+  Divider,
+  Grid,
+  IconButton,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography
+} from '@mui/material'
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined'
+import EditIcon from '@mui/icons-material/Edit'
+import DeleteIcon from '@mui/icons-material/Delete'
+import {makeStyles} from '@mui/styles'
+import React from 'react'
+import {useNavigate} from 'react-router-dom'
+import {Channel, ChannelRoute, Routes} from '../../../types'
+import {ChannelRoute as ChannelRouteComponent} from './ChannelRoute'
+import {useBasicDialog} from '../../../contexts/dialog.context'
+import {useConfirmation} from '../../../contexts/confirmation.context'
+
+const useStyles = makeStyles(_theme => ({
+  divider: {
+    marginTop: '10px',
+    margin: '0px',
+    width: '100%',
+    marginBottom: '10px',
+    overflow: 'visible'
+  },
+  divider2: {
+    marginTop: '10px',
+    margin: '0px',
+    width: '100%',
+    marginBottom: '30px',
+    overflow: 'visible'
+  },
+  addNewRouteContainer: {
+    padding: '10px'
+  },
+  emptyState: {
+    padding: '20px',
+    textAlign: 'center',
+    color: '#999',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '200px'
+  },
+  emptyStateIcon: {
+    fontSize: '48px',
+    marginBottom: '10px',
+    color: '#999'
+  }
```

Comment:
Why don't we move this to a css file

Reply:
To take advantage of TypeScript and that only works if the styles are defined in a .ts file.

I think perhaps I can move it to a separate ts file.